options.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head><title>My Test Extension Options</title></head>
```

Comment:
Would you update the title of this page?
```suggestion
<head><title>RUM Demo Extension Options</title></head>
```